### ISSUE TITLE
cannot find NULL values

### ISSUE THREAD

**Issue by Kharhamel** [thecodingmachine/database.tdbm#75]
select \* from 'table' where 'col' is null    never return anything

**Comment by moufmouf** [comment]
Ok, I was in the process of fixing this in the SQL parser.
Basically, what I was doing is transorming:

```
select * from 'table' where 'col' = :col
```

into

```
select * from 'table' where 'col' IS NULL
```

if `"col" === null`

Now, this is a bit cheating. And I found out that MySQL has a <=> (null-safe equals operator): https://dev.mysql.com/doc/refman/5.5/en/comparison-operators.html#operator_equal-to

So the real correct way of writing this would be to use:

```
select * from 'table' where 'col' <=> :col
```

@nguyenk What do you think? Should I still transform a "= null" into a "IS null" or is this the responsibility of the developer to use the correct "<=>" operator?

(basically, should I assume that everybody is wrong and fix the error, or should I force developers into discovering the  <=> operator?)

**Comment by moufmouf** [comment]
Fixed in MagicQuery 1.1.1
